docs/machine_learning/3_evaluation.md

```diff
@@ -0,0 +1,195 @@
+# Model Evaluation
+
+Before deploying a model to be consumed by several applications,
+we need to know what is the expected performance that it will achieve.
+After training a model using the collected and transformed dataset,
+concrete metrics have to be calculated so we can have a vision of
+how well the algorithm can model the data that we have on hand.
+
+Depending on the type of the problem (supervised, unsupervised,
+classification, regression), there are different metrics
+to evaluate. In supervised learning, we have available target values
+that we can use to compare to what was predicted by the model.
+However, the metrics are different between classification and regression problems
+since in the first one the target variable is not continuous.
+
+Another topic is worth mentioning, and that can lead to misleading
```

Comment:
"Another topic is worth mentioning, and that can lead to misleading conclusions is what data should be used to evaluate."

Acho que assim ficava mais claro:

"Another topic worth mentioning, that can lead to misleading conclusions, is what data should be used to evaluate."